packages/cardano-services/test/StakePool/DbSyncStakePoolProvider/__snapshots__/StakePoolBuilder.test.ts.snap

```diff
@@ -1518,6 +1518,10 @@ Array [
 
 exports[`StakePoolBuilder queryPoolStats returns active, retired and retiring pools count 1`] = `
 Object {
+  "averages": Object {
+    "apy": 0.0000158041676290942,
+    "margin": 0.000201057770545159,
```

Comment:
This value seems too low, but could be due to the data snapshot. We need to verify that it is a realistic number. cc @iadmytro

Reply:
Yep, this is due to the db snapshot, there are many missing rows of `pool_update` table. So, a lot of pools have margin = 0